python/rmm/_cuda/gpu.py

```diff
@@ -78,13 +79,12 @@ def runtimeGetVersion():
     The version is returned as (1000 major + 10 minor). For example,
     CUDA 9.2 would be represented by 9020.
 
-    This function automatically raises CUDARuntimeError with error message
-    and status code.
+    This calls numba.cuda.runtime.get_version() rather than cuda-python due to
+    current limitations in cuda-python.
     """
-    status, version = cudart.cudaRuntimeGetVersion()
-    if status != cudart.cudaError_t.cudaSuccess:
-        raise CUDARuntimeError(status)
-    return version
+
+    major, minor = numba.cuda.runtime.get_version()
```

Comment:
```suggestion
    # TODO: Replace this with `cudart.cudaRuntimeGetVersion`
    # call once Universal runtime is implemented
    major, minor = numba.cuda.runtime.get_version()
```

Looks good, shall we also have a TODO?